src/server.js

```diff
@@ -55,3 +61,7 @@ hypernova.createGetComponent = createGetComponent;
 hypernova.createVM = createVM;
 hypernova.getFiles = getFiles;
 hypernova.loadModules = loadModules;
+hypernova.worker = worker;
+hypernova.logger = logger;
+hypernova.defaultConfig = defaultConfig;
+hypernova.raceTo = raceTo;
```

Comment:
do all these need to be exported as properties, when they can be deep-imported from their original paths?

Reply:
Just following the pattern of all the things above it that could also be deep-imported.  This feels a bit less vulnerable to internal changes in hypernova.